pages/wg/outreach.md

```diff
@@ -20,77 +20,110 @@ bi-weekly virtual meetings. To get involved, visit the
-Feather RSE panel. If you are interested in participating as a panel member,
-please fill out [this form](https://forms.gle/PUcfcD7LnKVXnZfJ8).
+28 winners were selected to receive tumblers and backpacks, but all entrants
+were sent official US-RSE stickers.
 
-### Customizable RSE Introduction Template
+## Outreach Material Cultivation
 
-Many members of US-RSE have spoken in diverse environments to a variety of
-audiences about US-RSE. There has been no standardized presentation or
-template for ad-hoc events such as these.
+Many US-RSE members have already conducted outreach as part of their daily lives.
+To promote reusability, the Outreach Working Group has created a
+[US-RSE Outreach Materials Google Drive](https://drive.google.com/drive/folders/1gu_0fhAZt7jjrDFWTHekyRLBboCZUnc8)
+where anyone can submit their materials, articles, blog posts, etc., for use
+and reference by other US-RSE members.
 
-To address this gap, the Outreach working group
-aims to create an "Ad-hoc US-RSE Overview Template" that is customizable for
-different audiences but provides a basic set of standardized content.
+Please submit your own content in the appropriate folder and list it in the
+[main manifest document](https://docs.google.com/spreadsheets/d/1TOl1ApTvzKXeuO38ZSsrShNXN1uTU9AgetlFMJexKuk/edit?usp=sharing)!
 
-### Career Counseling
+## Rapid Interviews
 
-As Research Software Engineering is still a relatively new professional
-designation, the Outreach working group would like to broaden awareness of
-paths that lead to RSEs throughout universities, colleges, and high schools.
+We are actively recruiting US-RSE members to submit short self-recorded interviews
+for use on our [website](https://us-rse.org) and at other events, such as Supercomputing 2022.
 
-We aim to create connections with career counseling programs and provide
-resources about what RSEs are and how to become an RSE such that the younger
-generation is introduced to the concept sooner in their educational careers.
+We ask that interested members answer a short series of questions, including:
 
-### Member Employer Outreach and Advocacy
+- Tell us about you - where do you live, where do you work, what are your interests, what do you currently work on?
+- What are you working on right now that is exciting?
+- How did you hear about RSEs and the US-RSE?
+- Why did you join the US-RSE?
+- Describe your work as an RSE or RSE ally in one sentence.
+- Where would you like to see the RSE profession in 10 years?
 
-Many US-RSE members' employers do not have a clear understanding of the RSE
-profession and a clear growth path for their employed RSEs. We hope to encourage
-outreach to member employers in order to broaden their understanding of RSE
-needs.
+If you are interested in submitting a rapid interview, please refer to our
+[instruction guide](https://docs.google.com/document/d/1Q_D1mf8h5ryqy4y73j16g2-HY7wp-pBsIQj7O6k9u4s/edit?usp=sharing).
```

Comment:
I'd really love to say, if you do it, we'll send you some stickers in the mail. I just wish it wasn't me signing up for more work :).